CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 17.9.2
+
+Java 8 on App Engine standard environment is now generally available.
```

Comment:
Maybe link the "generally available" text to the blogpost: https://cloudplatform.googleblog.com/2017/09/Java-8-on-App-Engine-Standard-environment-is-now-generally-available.html

Reply:
good idea. done - ptal